### ISSUE TITLE
AUR support

### ISSUE THREAD

**Issue by System64fumo** [silime/kernel-builder#1]
Hello, I randomly stumbled uppon this repo and struck gold!
It would be nice to have these packages on the AUR for easier access.

So could kindly you put these on the AUR?
If you can't or don't wish to i'd be more than happy to maintain a few.

**Comment by silime** [comment]
Sure, that’s something I’ve been meaning to do. I just haven’t had the time to put them on the AUR. It’s great that you’re interested in doing that.

**Comment by System64fumo** [comment]
Awesome!
With your permission could i maintain the alsa ucm and iio sensor proxy packages?

As for the linux firmware part, Shouldn't it only provide the firmware for sdm845 and not the whole linux firmware?

**Comment by silime** [comment]
The wcn3390 firmware in the linux-firmware package needs to be replaced with a OnePlus signed firmware, which may cause conflicts. I don't know how to fix it, so I repackaged linux-firmware, and I'm going to keep the three directories of qcom qca ath10k under linux-firmware to reduce unnecessary firmware?

**Comment by silime** [comment]
Yes, you can maintain the  alsa ucm and iio sensor proxy packages. Additionally, you are welcome to maintain any other packages in the repository if they are useful to you.

**Comment by System64fumo** [comment]
Honestly didn't even know you needed a oneplus signed firmware?
WiFi worked fine for me excluding the constant errors in dmesg about keys or whatever every 30 seconds iirc

As for the firmware, why not just add the override to /sys/class/firmware/firmware_class/path before loading the modules or add a fw_dir= cmdline?

**Comment by silime** [comment]
WIFI and Bluetooh worked fine without oneplus signed board-2.bin and crnv21.bin firmware? I haven't tested it yet, so just need the linux-firmware-oneplus-sdm845-qcom package to provide the firmware for the sdm845?

**Comment by System64fumo** [comment]
Actually i just remembered that i don't use linux-firmware but instead copied the whole firmware dir from postmarket os, oddly enough Bluetooth is inconsistent, breaks every once and a while

**Comment by System64fumo** [comment]
By the way with iio-sensor-proxy what do i name the package?
ssc-sensor-proxy? or iio-sensor-proxy-libssc? Or something else?

**Comment by System64fumo** [comment]
@silime Bad news, I put an unrelated package on the AUR yesterday (aarch64 only) and i've just received an email telling me that it's going to get deleted.

Apparently AUR packages MUST work on x86_64 arch linux
So any exclusive arm content is not allowed

So you either have to make your arm programs work with x86_64 or not have them on the AUR

This is deeply saddening as i'm a huge ARM advocate.
